tools/envoy_headersplit/requirements.txt

```diff
@@ -0,0 +1 @@
+clang==6.0.0.2
```

Comment:
This seems a very old version of Clang if the pip version is that of the toolchain..

Reply:
now it's set as 10.0.1